packages/mdc-select/_mixins.scss

```diff
@@ -221,12 +217,14 @@
 
 @mixin mdc-select-floating-label_ {
   .mdc-floating-label {
-    @include mdc-rtl-reflexive-position(left, $mdc-select-outline-label-offset);
-
     bottom: 12px;
     line-height: 1.75rem;
     pointer-events: none;
   }
+
+  &:not(.mdc-select--outlined) .mdc-floating-label {
+    @include mdc-rtl-reflexive-position(left, $mdc-select-outline-label-offset);
```

Comment:
Should this also be using the notched-outline-leading-width variable? It's unclear to me why this is using a different value (16px), and also why it's using a variable named for outlined select, in a selector that's specifically not outlined select.